photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimatorEvaluationFunction.scala

```diff
@@ -34,7 +33,8 @@ class GameEstimatorEvaluationFunction(
     estimator: GameEstimator,
     baseConfig: GameOptimizationConfiguration,
     data: DataFrame,
-    validationData: DataFrame)
+    validationData: DataFrame,
+    range: DoubleRange)
```

Comment:
Can you add a documentation here explaining the different parameters? Especially if the range in [0.001, 1000] or [-3, 3]. I feel the range here should be [-3,3]

Reply:
Yes, range is [-3, 3]. Documentation will be added in the code.